### DISCUSSION TITLE
Multiple contacts

### DISCUSSION THREAD

**Discussion by FELS-Zak** [kimai/kimai#4193] [Questions & Answers]
Is it possible to add multiple contacts to a client account when you're creating the client ?

**Comment by lucas-0liveira** [comment]
@FELS-Zak  Yes, absolutely! When you're setting up a client account, you can definitely add multiple contacts associated with that client. This way, you'll have a comprehensive record of everyone involved with the client's account, making it easier to communicate and coordinate with them in the future

**Reply by kevinpapst** [reply]
Ignore this ChatGPT generated non-sense, user will be removed immediately.
